bin/reth/src/node/mod.rs

```diff
@@ -11,7 +11,7 @@ use clap::{crate_version, Parser};
 use eyre::Context;
 use fdlimit::raise_fd_limit;
 use futures::{stream::select as stream_select, Stream, StreamExt};
-use reth_consensus::beacon::BeaconConsensus;
+use reth_consensus::beacon::BeaconConsensusBuilder;
```

Comment:
I'd like to use all builders like `BeaconConsensus::builder()` instead of importing the builder